src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/mediation/MediationFailedCallbackHandler.java

```diff
@@ -100,6 +104,10 @@ private CallbackResponse notifyPartiesOfOutcome(CallbackParams callbackParams) {
             .trackEvent(getAppInsightEventBasedOnMediationPilot(claim), REFERENCE_NUMBER, claim.getReferenceNumber());
 
         notificationService.notifyParties(claim);
+
+        if (!FeaturesUtils.isOnlineDQ(claim)) {
```

Comment:
This is not behind feature toggle

Reply:
I think this was existing feature

Reply:
within this **if** statement there is **updateCaseEvent** which wasn't there before. Looks like we are updating case event now

Reply:
this change is not for CTSC, it was one flow of failed mediation event